packages/semcom-node/lib/component/services/component-transformer.service.spec.ts

```diff
@@ -17,4 +39,66 @@ describe('ComponentTransformerService', () => {
 
   }));
 
+  describe('toQuads()', () => {
+
+    it('transforms componentMetadata into a quad array', () => {
+
+      const quads = transformer.toQuads(componentMetaDatas);
+
+      componentMetaDatas.forEach((component) => {
+
+        expect(quads).toContainEqual(Quad(
+          component.uri,
+          'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',
+          'http://semcom.digita.ai/voc#component',
+        ));
+
+        [ 'label', 'description', 'author', 'tag', 'version', 'latest' ].forEach((predicate) => {
```

Comment:
Maybe this better be something like `keyof ComponentMetadata`, to be sure this test fails when someone adds a field there?

Reply:
Also: aren't both length tests already sumbsumed by this one?

Reply:
the length test confirms `quads` doesn't contain more quads than it should have